### DISCUSSION TITLE
MariaDB?

### DISCUSSION THREAD

**Discussion by robertsilen** [umami-software/umami#2943] [General]
Does umami work with MariaDB?

**Comment by FHU-yezi** [comment]
It should work because Umami does not use any MySQL-only feature.

And maybe you will get some preformance benfits as MariaDB use more advanced query optimizer.

**Comment by robertsilen** [comment]
Great! Do you mind if I suggest a PR to add a mention of MariaDB to the README? It currently mentions MySQL 8.0, what version would be appropriate for MariaDB?

**Comment by Amerlander** [comment]
Actually, I was using MariaDB for a while, but changed to MySQL, when there was an update with a migration using `BIN_TO_UUID()` which is not available in MariaDB (`05_add_visit_id`).

So I would not recomend using MariaDB.

**Reply by robertsilen** [reply]
Yes, indeed, that difference between e.g. MySQL 8.0 and MariaDB 11.4 is confirmed by [https://mariadb.com/kb/en/function-differences-between-mariadb-11-4-and-mysql-8-0/#uuid](https://mariadb.com/kb/en/function-differences-between-mariadb-11-4-and-mysql-8-0/#uuid). I now linked to this discussion from MariaDB's corresponding Jira item [MDEV-15854](https://jira.mariadb.org/browse/MDEV-15854).
